mantidproject/newstarter
# Add EX01 & EX02 Solution

**Description of work.**
Created solution to Ex01, allows counting and sorting of words in a txt file. 

**To test:**
1. Build and run the main method. 
2. Supply an input file path
3. Supply an output file path
4. Check output file for errors and counts

*There is no associated issue.*


*This does not require release notes* because **it is a new starter exercise.**

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
